journal-of-investigative-dermatology.csl

```diff
@@ -1,227 +1,309 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-US">
-  <!-- This style was edited with the Visual CSL Editor (http://steveridout.com/csl/visualEditor/) -->
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Journal of Investigative Dermatology</title>
+    <title>Journal of investigative dermatology</title>
+    <title-short>J Invest Dermatol</title-short>
     <id>http://www.zotero.org/styles/journal-of-investigative-dermatology</id>
     <link href="http://www.zotero.org/styles/journal-of-investigative-dermatology" rel="self"/>
-    <link href="http://www.zotero.org/styles/ecology-letters" rel="template"/>
-    <link href="http://www.nature.com/jid/author_instructions.html" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="http://endnote.com/downloads/style/journal-investigative-dermatology" rel="documentation"/>
+    <link href="http://www.jidonline.org/content/authorinfo" rel="documentation"/>
     <author>
-      <name>Sebastian Karcher</name>
+      <name>Charles Parnot</name>
+      <email>charles.parnot@gmail.com</email>
+      <uri>http://twitter.com/cparnot</uri>
+    </author>
+    <author>
+      <name>Max Gordon</name>
+      <email>max@gforge.se</email>
+      <uri>http://gforge.se</uri>
     </author>
     <category citation-format="author-date"/>
     <category field="biology"/>
-    <issn>0022-202X</issn>
     <eissn>1523-1747</eissn>
-    <updated>2012-09-14T21:22:32+00:00</updated>
+    <issnl>0022-202X</issnl>
```

Comment:
we don't tend to use issnl anymore, since it never really went anywhere. Leave this as simple issn.